.github/actions/spelling/expect.txt

```diff
@@ -138,7 +142,6 @@ MDBs
 methoddeclaration
 micronaut
 Migratetohypersistenceutilshibernate
-Migratetohypersistenceutilshibernate6
```

Comment:
check-spelling will now complain about things like this (expect only wants words, and by default numbers can't be in words -- although with this version you could actually change the definition of a word...)